CMakeLists.txt

```diff
@@ -34,15 +34,14 @@ else()
     set(BOOST_URL_BUILD_TESTS ${BUILD_TESTING})
 endif()
 
-
 include(GNUInstallDirs)
 if(BOOST_URL_IS_ROOT)
     if (NOT EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/../../CMakeLists.txt")
         set(BOOST_URL_FIND_PACKAGE_BOOST ON)
     endif()
 
     # container comes from json
-    set(BOOST_INCLUDE_LIBRARIES url json)
+    set(BOOST_INCLUDE_LIBRARIES url json filesystem)
```

Comment:
Does this make Boost.URL require filesystem to build? Or is it just for the examples?

Reply:
Just the examples. Since the URL is not allowed to access the parent path. maybe we could remove `filesystem` from the examples too and still make the point about decoding and appending to a string that represents a path.

Reply:
Exploring the interaction between filesystem and URL is important, so we should keep it using `filesystem::path`.